controllers/replicationsource_test.go

```diff
@@ -463,6 +464,35 @@ var _ = Describe("ReplicationSource", func() {
 		})
 	})
 
+	Context("when a port is defined", func() {
+		BeforeEach(func() {
+			remotePort := int32(2222)
+			remoteAddr := "my.remote.host.com"
+			rs.Spec.Rsync = &scribev1alpha1.ReplicationSourceRsyncSpec{
+				ReplicationSourceVolumeOptions: scribev1alpha1.ReplicationSourceVolumeOptions{
+					CopyMethod: scribev1alpha1.CopyMethodClone,
+				},
+				Address: &remoteAddr,
```

Comment:
@JohnStrunk I'm having issues here with having both address and Port defined. 
```
1 chart(s) linted, 0 chart(s) failed
/home/rcook/go/bin/golangci-lint run ./...
controllers/replicationsource_test.go:476: File is not `goimports`-ed (goimports)
				Port: &remotePort,
make: *** [Makefile:42: test] Error 1
```

Reply:
Looks like it's a formatting issue, not syntax:
```
$ goimports -d controllers/replicationsource_test.go
diff -u controllers/replicationsource_test.go.orig controllers/replicationsource_test.go
--- controllers/replicationsource_test.go.orig	2021-03-22 15:00:15.260044777 -0400
+++ controllers/replicationsource_test.go	2021-03-22 15:00:15.260044777 -0400
@@ -473,7 +473,7 @@
 					CopyMethod: scribev1alpha1.CopyMethodClone,
 				},
 				Address: &remoteAddr,
-				Port: &remotePort,
+				Port:    &remotePort,
 			}
 		})
 		It("an environment variable is created", func() {
```

goimports (kind of) supersedes gofmt, so it requires standard go formatting of whitespace.